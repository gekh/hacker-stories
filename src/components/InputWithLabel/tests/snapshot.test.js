import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import { InputWithLabel } from '..';


describe('InputWithLabel', () => {
  it('renders a snapshot', () => {
    const { container } = render(<InputWithLabel
      id="search"
      value="React"
      onInputChange={jest.fn()}
    >
      Search:
    </InputWithLabel>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
