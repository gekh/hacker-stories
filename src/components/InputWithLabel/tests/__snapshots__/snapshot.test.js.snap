// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWithLabel renders a snapshot 1`] = `
<label
  class="sc-bcXHqe cNyZAq"
  for="search"
>
  Search:
</label>
`;
